### PR TITLE
#108 possible fix

### PR DESCRIPTION
**Pull request by aethelwulffe**
I have not had the existing code fail, but...I ain't using the cutting
edge of the butter knife.  Try these wrapper tags.

### CODE REVIEW

File: sql/0_0_1-to-1_0_0_upgrade.sql [hunk 1]
```diff
@@ -1624,14 +1624,38 @@ INSERT INTO list_options (list_id,option_id,title) VALUES ('lists','issue_subtyp
...
+#IfTable openemr_postcalendar_topics
 RENAME `openemr_postcalendar_topics` TO `libreehr_postcalendar_topics`;
+#EndIf
+
+#IfTable openemr_session_info
 RENAME `openemr_session_info` TO `libreehr_session_info`;
+#EndIf
+
+#IfTable openemr_module_vars
 RENAME `openemr_module_vars` TO `libreehr_module_vars`;
```
Comment (aethelwulffe) [inline sql/0_0_1-to-1_0_0_upgrade.sql]: Oops.  Right, move up high...  Going for try 3

File: sql/0_0_1-to-1_0_0_upgrade.sql [hunk 2]
```diff
@@ -71,6 +71,38 @@
 --  #IfNotListOccupation
 --    Custom function for creating Occupation List
 
+#IfTable
+RENAME `openemr_module_vars` TO `libreehr_module_vars`;
```
Comment (tmccormi) [inline sql/0_0_1-to-1_0_0_upgrade.sql]: Doesn;t the syntax require the word TABLE?   As in
RENAME TABLE `openemr_module_vars` TO `libreehr_module_vars`;

Comment (tmccormi) [inline sql/0_0_1-to-1_0_0_upgrade.sql]: http://dev.mysql.com/doc/refman/5.7/en/rename-table.html

**Comment by aethelwulffe** [comment]
OK, this "jump on something then push then jump on something else then..." stuff isn't working.  I think I may turn off my phone.  Try4 seems to actually be complete.  I don't think there are any collisions with the rename at the head of the file now.  SED (or CE as it is on my machine) can be used for great good or great harm....

**Pull request merged**